src/CosmosDB/CosmosDB.Sdk/README.md

```diff
@@ -0,0 +1,41 @@
+# Overall
+This directory contains the service clients of Az.CosmosDB module.
+
+## Run Generation
+In this directory, run AutoRest:
+```
+autorest.cmd README.md --version=v2
+```
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+csharp: true
+clear-output-folder: true
+reflect-api-versions: true
+openapi-type: arm
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+
+title: CosmosDBManagementClient
+```
+
+
+### 
+``` yaml 
+input-file:
+  - https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-08-15-preview/cosmos-db.json
+  - https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-08-15-preview/dataTransferService.json
+  - https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-08-15-preview/managedCassandra.json
+  - https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-08-15-preview/mongorbac.json
+  - https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-08-15-preview/notebook.json
+  - https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-08-15-preview/rbac.json
```

Comment:
The link of input-file should be permalink. Otherwise, it won't be easy to find which file is used for code generation when  file is changed.